examples/cluster_queues/pause/main.go

```diff
@@ -35,7 +35,7 @@ func main() {
 	resp, err := client.ClusterQueues.Pause(*org, *clusterID, *queueID, &clusterQueuePause)
 
 	if err != nil {
-		log.Fatalf("Pausing dispatch on cluster queue %s failed: %s", *clusterID, err)
+		log.Fatalf("Pausing dispatch on cluster queue %s failed: %s", *queueID, err)
```

Comment:
Assuming this is a fix from the queue PR that was merged.

Reply:
Sure is 😉